airbyte-integrations/connectors/source-orb/integration_tests/configured_catalog.json

```diff
@@ -38,28 +38,24 @@
     },
     {
       "stream": {
-        "name": "credits_ledger_entries",
+        "name": "invoices",
         "json_schema": {},
         "supported_sync_modes": ["full_refresh", "incremental"],
         "source_defined_cursor": true,
-        "default_cursor_field": ["created_at"],
+        "default_cursor_field": ["invoice_date"],
         "source_defined_primary_key": [["id"]]
       },
       "sync_mode": "incremental",
       "destination_sync_mode": "overwrite"
     },
     {
       "stream": {
-        "name": "subscription_usage",
```

Comment:
why are you removing this stream from the integration tests configured catalog?

Reply:
Great catch - I'd just removed this for the purpose of running the integration test more quickly as noted above since I didn't modify this stream, but I'll add it back in!